src/core/theme.js

```diff
@@ -19,9 +19,12 @@ export const theme = {
     medium: 767,
     large: 1024,
     extraLarge: 1199,
+    bigest: 1389,
```

Comment:
It is good idea, but it should be "biggest" or "theBiggest" as it is in English.